apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -51,17 +56,17 @@ function getParamsForAssessment(
   const assessment = assessmentInfoFile.data;
   if (!assessment) throw new Error(`Missing assessment data for ${assessmentInfoFile.uuid}`);
 
-  const allowIssueReporting = assessment.allowIssueReporting ?? true;
-  const allowRealTimeGrading = assessment.allowRealTimeGrading ?? true;
-  const requireHonorCode = assessment.requireHonorCode ?? true;
-  const allowPersonalNotes = assessment.allowPersonalNotes ?? true;
+  const allowIssueReporting = assessment.allowIssueReporting;
+  const allowRealTimeGrading = assessment.allowRealTimeGrading;
+  const requireHonorCode = assessment.requireHonorCode;
+  const allowPersonalNotes = assessment.allowPersonalNotes;
 
   // It used to be the case that assessment access rules could be associated with a
   // particular user role, e.g., Student, TA, or Instructor. Now, all access rules
   // apply only to students. So, we filter out (and ignore) any access rule with a
   // non-empty role that is not Student.
-  const allowAccess = (assessment.allowAccess ?? [])
-    .filter((accessRule) => !('role' in accessRule) || accessRule.role === 'Student')
+  const allowAccess = assessment.allowAccess
+    .filter((accessRule) => accessRule.role == null || accessRule.role === 'Student')
```

Comment:
I switched to a consistent `X == null` or `X != null` everywhere in `fromDisk` since the parse process could now have emitted values that were defaulted to null.